.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,48 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines: https://git.k8s.io/community/contributors/guide/first-contribution.md#your-first-contribution and developer guide https://git.k8s.io/community/contributors/devel/development.md#development-guide
```

Comment:
Also, let's have a link for https://github.com/kubernetes-sigs/kube-scheduler-wasm-extension/blob/main/CONTRIBUTING.md.

It's now a default one tho, we can improve later.

Reply:
Copied from the k/k repo, just removed the last part about the docs.